AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json

```diff
@@ -4,7 +4,6 @@
  "categories": ["Toolkit/oneAPI AI And Analytics/AI Getting Started Samples"],
  "description": "This sample illustrates how to run ILIT to quantize the FP32 model trained by Keras on Tensorflow to INT8 model to speed up the inference.",
  "languages": [{"python":{}}],
- "dependencies": ["tensoflow","notebook","matplotlib"],
  "os": ["linux"],
```

Comment:
This should be 'tensorflow' and 'iLiT'. I was just creating a PR for this but can you please edit this one.

Reply:
ok. got it.

Reply:
Sorry to ping again but the letter 'r' is missing from tensorflow word.

Reply:
MErging this, will fix the r directly.

Reply:
r is fixed.